src/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Http\Requests\LoginUserRequest;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+
+class LoginController extends Controller
+{
+    public function formPage()
+    {
+        return view("user.login");
```

Comment:
This is not working. Case sensitive in folder name is important (`resources/views/User/login.blade.php`).

![obraz](https://user-images.githubusercontent.com/29115024/156580493-320964ce-073c-4743-99e1-d6a440a6ea9a.png)

Reply:
I would prefer small letters in names of view folder .